mlflow/genai/judges/utils.py

```diff
@@ -287,7 +721,10 @@ def _invoke_judge_model(
         num_completion_tokens = output.num_completion_tokens
     elif _is_litellm_available():
         # prioritize litellm for better performance
-        response = _invoke_litellm(provider, model_name, prompt, num_retries)
+        from mlflow.types.llm import ChatMessage
+
+        messages = [ChatMessage(role="user", content=prompt)]
+        response = _invoke_litellm(provider, model_name, messages, None, num_retries)
```

Comment:
Nit - can we specify keyword arg names for this _invoke_litellm call so that it's more obvious why `None` is being passed? (`trace=None`)?